frontend/src/routes/Home/Overview/OverviewPage.tsx

```diff
@@ -418,40 +466,64 @@ export default function OverviewPage() {
     ]
   }, [cloudLabelFilter, offline, ready, t])
 
+  const clusterAddonData = useMemo(() => {
+    return [
+      {
+        key: t('Degraded'),
+        value: danger.count,
+        link: buildClusterLinks(Array.from(danger.clusters), 'ClusterManagementAddOn'),
+      },
+      {
+        key: t('Progressing'),
+        value: progress.count,
+        link: buildClusterLinks(Array.from(progress.clusters), 'ClusterManagementAddOn'),
+      },
+      {
+        key: t('Pending'),
+        value: pending.count,
+        link: buildClusterLinks(Array.from(pending.clusters), 'ClusterManagementAddOn'),
+      },
+      {
+        key: t('Unknown'),
+        value: unknown.count,
+        link: buildClusterLinks(Array.from(unknown.clusters), 'ClusterManagementAddOn'),
+      },
+      {
+        key: t('Available'),
+        value: healthy.count,
+        isPrimary: true,
+        link: buildClusterLinks(Array.from(healthy.clusters), 'ClusterManagementAddOn'),
+      },
+    ]
+  }, [healthy, danger, progress, pending, unknown, t])
+
   const policyReportData = useMemo(() => {
+    const getLink = (type: any, count: number) => {
+      return `${NavigationPath.search}?filters={"textsearch":"kind%3APolicyReport%20${type}%3A${
+        count > 0 ? '>' : '='
```

Comment:
I don't think we should change the query depending on the current count. The tile says "there are 0 clusters with critical issues" not "these are the clusters with 0 critical issues".

Reply:
cool--what should the query be?

Reply:
Just the version with `>0` always.